myst_nb/parser.py

```diff
@@ -230,7 +230,7 @@ def render_nb_code_cell(self, token: Token):
         # ==================
         # Cell output
         # ==================
-        if "remove_output" not in tags:
+        if "remove_output" not in tags and cell["outputs"]:
```

Comment:
Here I now skip the creation of the `CellOutputNode` if the code cell has no outputs. @choldgraf any reason you can think of why this should not be done?

Reply:
Nah that sounds like a good idea to me

Reply:
👍